plugin-server/src/main/ingestion-queues/batch-processing/each-batch-ingestion.ts

```diff
@@ -61,22 +60,10 @@ export async function ingestEvent(
 
     checkAndPause?.()
 
-    // Eventually no events will have a team_id as that will be determined and
-    // populated in the plugin server rather than the capture endpoint. However,
-    // we support both paths during the transitional period.
-    if (event.team_id) {
-        server.statsd?.increment('kafka_queue_ingest_event_hit', {
-            pipeline: 'runEventPipeline',
-            team_id: event.team_id.toString(),
-        })
-        // we've confirmed team_id exists so can assert event as PluginEvent
-        await workerMethods.runEventPipeline(event as PluginEvent)
```

Comment:
non-blocking

Is this called anywhere else now? Maybe not for this PR but how easy would it be to remove altogether? Might be a pain with all the test references.

Reply:
I tried to, but there's indeed a lot of test references everywhere. I have a branch dropping `runEventPipeline` and renaming `runLightweightCaptureEndpointEventPipeline` to `runEventPipeline`, but that branch has a lot of test breakages.

Want to get that code in prod fast, to confirm out hypothesis, while I cleanup the other branch,

Reply:
Draft PR for the cleanups: https://github.com/PostHog/posthog/pull/14365